src/OpenDebugAD7/VariableManager.cs

```diff
@@ -14,9 +14,21 @@ internal class VariableEvaluationData
         internal enum_DEBUGPROP_INFO_FLAGS propertyInfoFlags;
     }
 
+    internal enum VariablesScope
+    {
+        Locals,
+        Registers
+    }
+
+    internal class VariablesRef
+    {
+        internal IDebugStackFrame2 StackFrame;
+        internal VariablesScope Scope;
+    }
```

Comment:
was a preliminary name/solution, feel free to suggest something better

Reply:
I might suggest calling this 'Scope' and the enum 'ScopeKind'. If 'Scope' creates name conflicts, this could also be 'ScopeState'

---
In reply to: [493603479](https://github.com/microsoft/MIEngine/pull/1039#discussion_r493603479) [](ancestors = 493603479)